static/recruitment/index.html

```diff
@@ -0,0 +1,179 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Fraternity page for Alpha Epsilon of Phi Kappa Sigma">
+
+    <title>Fraternity &ndash; Alpha Epsilon of Phi Kappa Sigma</title>
+
+    <!-- Import Pure components -->
+    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+
+    <!--[if lte IE 8]>
+
+        <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-old-ie-min.css" integrity="sha384-sGrAmFbC85ibuG+7mNPSCxr6U5S3ObASQDpRTlck+CBI4SRyHG9lIto9YyA92YG9" crossorigin="anonymous">
+
+    <![endif]-->
+    <!--[if gt IE 8]><!-->
+
+        <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css" integrity="sha384-b92sF+wDNTHrfEtRaYo+EpcA8FUyHOSXrdxKc9XB9kaaX1rSQAgMevW6cYeE5Bdv" crossorigin="anonymous">
+
+    <!--<![endif]-->
+
+    <!-- Import other components -->
+
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.css">
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400italic,400,700' rel='stylesheet' type='text/css'>
+
+    <!-- Import css -->
+    <link rel="stylesheet" href="../css/layouts/aepks.css">
```

Comment:
Might want to add a diff background image. Maybe Haney has something, or could grab an old rush photo from the drive